distribution/docs/src/main/resources/content/_configuring/http-port.adoc

```diff
@@ -1,17 +1,23 @@
-:title: Configuring HTTP Port
+:title: Configuring HTTP(s) Ports
```

Comment:
❓ Should the `s` be capitalized here and in the summary?

Reply:
yes. fixed.